include/pdmatrix.hh

```diff
@@ -26,6 +26,8 @@ private:
 public:
     Matrix<T>(std::initializer_list<T> l, size_t rows, size_t cols): matrix_(l), rows_(rows), cols_(cols){};
 
+    Matrix<T>(Matrix<T>& m):matrix_(m.matrix_), rows_(m.rows()),cols_(m.cols()){};
```

Comment:
Copy constructor must accept const ref as parameter

Reply:
check if it's right now